it/flags-provider/src/test/java/com/linecorp/armeria/common/BaseFlagsProvider.java

```diff
@@ -71,6 +71,11 @@ public Long defaultServerConnectionDrainDurationMicros() {
         return 500L;
     }
 
+    @Override
+    public Long defaultClientConnectionDrainDurationMicros() {
+        return 0L;
```

Comment:
I've set this to `0` for backward compatibility(didn't want to set user's default to utilize connection duration)

Reply:
Without this, a test fails in this IT module?

Reply:
Because I've added `DEFAULT_CLIENT_CONNECTION_DRAIN_DURATION_MICROS` as 0L in `DefaultFlagsProvider`, I think we cam omit it.